docker-hub/api/dvp.yaml

```
@@ -490,9 +666,12 @@ components:
     TimespanType:
       type: string
       enum: [months,weeks]
+    PeriodType:
+      type: string
+      enum: [last-2-months,last-3-months,last-6-months,last-12-months]
     DataviewType:
       type: string
-      enum: [raw,summary,repo-summary]
+      enum: [raw,summary,repo-summary,namespace-summary,geo-repo-summary]
```

geo aggregations are experimental and not yet released

```suggestion
      enum: [raw,summary,repo-summary,namespace-summary]
```